### ISSUE TITLE
Cannot add the plugin again after removing it

### ISSUE THREAD

**Issue by spinninghamster** [dpa99c/cordova-android-play-services-gradle-release#16]
**I'm submitting a ...**  (check one with "x"):
- [x] bug report
- [ ] feature request
- [ ] documentation issue

# Bug report

**Current behavior:**

I cannot re-add the plugin after removing it.

**Expected behavior:**

Remove the plugin with `cordova plugin remove cordova-android-play-services-gradle-release` and re-add it with `cordova plugin add cordova-android-play-services-gradle-release`

**Steps to reproduce:**

- I added the plugin.
- Then removed it.
- Verified with `cordova plugin list` that it's removed.
- Tried to add it again

**Environment information**

➜  app cordova -v
8.0.0
➜  app cordova platform ls
Installed platforms:
  android 7.0.0
  ios 4.5.5
Available platforms: 
  browser ~5.0.1
  osx ~4.0.1
  windows ~5.0.0
  www ^3.12.0
➜  app cordova plugin ls
com.googlemaps.ios 2.7.0 "Google Maps SDK for iOS"
cordova-android-play-services-gradle-release 1.4.3 "cordova-android-play-services-gradle-release"
cordova-open-native-settings 1.5.1 "Native settings"
cordova-plugin-buildinfo 2.0.2 "BuildInfo"
cordova-plugin-customurlscheme 4.3.0 "Custom URL scheme"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-dialogs 2.0.1 "Notification"
cordova-plugin-geolocation 4.0.1 "Geolocation"
cordova-plugin-google-analytics 1.8.6 "Google Universal Analytics Plugin"
cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
cordova-plugin-network-information 2.0.1 "Network Information"
cordova-plugin-request-location-accuracy 2.2.3 "Request Location Accuracy"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-plugin-x-toast 2.7.0 "Toast"
cordova.plugins.diagnostic 4.0.5 "Diagnostic"
onesignal-cordova-plugin 2.4.3 "OneSignal Push Notifications"
➜  app 

**Other information:**

```
cordova plugin add cordova-android-play-services-gradle-release --variable PLAY_SERVICES_VERSION=15.+
Installing "cordova-android-play-services-gradle-release" for android
Android Studio project detected
Error during processing of action! Attempting to revert...
Failed to install 'cordova-android-play-services-gradle-release': CordovaError: Uh oh!
"/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova-android-play-services-gradle-release/properties.gradle" already exists!
    at copyNewFile (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/lib/pluginHandlers.js:261:45)
    at install (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/lib/pluginHandlers.js:51:17)
    at ActionStack.process (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/cordova-common/src/ActionStack.js:56:25)
    at PluginManager.doOperation (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/cordova-common/src/PluginManager.js:114:20)
    at PluginManager.addPlugin (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/cordova-common/src/PluginManager.js:144:17)
    at /Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/Api.js:247:74
    at _fulfilled (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/q/q.js:854:54)
    at self.promiseDispatch.done (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/q/q.js:883:30)
    at Promise.promise.promiseDispatch (/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/q/q.js:816:13)
    at /Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova/node_modules/q/q.js:570:49
(node:9692) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): CordovaError: Uh oh!
"/Users/spinninghamster/Code/Live/Apps/app/platforms/android/cordova-android-play-services-gradle-release/properties.gradle" already exists!
(node:9692) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by spinninghamster** [comment]
I tried to manually remove the properties.gradle file (and folder) and it still doesn't work.

**Comment by spinninghamster** [comment]
The weird thing is, after this error message the plugin shows installed in `cordova plugin list`

**Comment by dpa99c** [comment]
Fix published to npm in v1.4.4
